.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.9.2
```

Comment:
shouldn't this be a symlink to `.python-version` to avoid confusion?

Reply:
No, the `.python-version` contains `3.9.2`, not `python 3.9.2`. The `.tool-versions` file is specifically for ASDF.